articles/faq/cancel-my-account.md

```diff
@@ -1,8 +1,18 @@
 ---
-title: "How Do I Cancel My Account?"
+title: "How Do I Downgrade or Cancel My Account?"
 description: "If you are a paying customer and you wish to cancel your account you can go into Admin Billing in your Spark Post account and click on the Free plan Your billing will automatically stop Please note If you have a customized pricing account please contact us..."
```

Comment:
If you are a paying customer and wish to cancel your account, you can go into Admin Billing

Reply:
updated the description, this was all old text actually.